integration/nwo/cmd/cryptogen/cryptogen.go

```diff
@@ -58,11 +57,17 @@ type NodeSpec struct {
 	StreetAddress      string   `yaml:"StreetAddress"`
 	PostalCode         string   `yaml:"PostalCode"`
 	SANS               []string `yaml:"SANS"`
+	HSM                bool     `yaml:"HSM"`
+}
+
+type UserSpec struct {
+	Name string `yaml:"Name"`
+	HSM  bool   `yaml:"HSM"`
 }
 
 type UsersSpec struct {
-	Count int      `yaml:"Count"`
-	Names []string `yaml:"Names"`
+	Count int        `yaml:"Count"`
```

Comment:
I think that for marshalling/unmarshalling we don't need the annotation for the public fields. Unless it's a convention we use in the project.